spec/support/idv_examples/cancel_at_idv_step.rb

```diff
@@ -19,20 +19,54 @@
     expect(current_path).to eq(original_path)
   end
 
-  it 'shows the user a cancellation message with the option to cancel and reset idv' do
-    click_link t('links.cancel')
+  context 'with an sp' do
+    it 'shows the user a cancellation message with the option to cancel and reset idv' do
+      failure_to_proof_url = 'https://www.example.com/failure'
+      sp_name = 'Test SP'
+      allow_any_instance_of(SessionDecorator).to receive(:failure_to_proof_url).
```

Comment:
This line and the one on line 30 can be removed by using the RSpec conditional filter I mentioned below. In this case, it would be `if: sp`.